pom.xml

```diff
@@ -11,6 +11,36 @@
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+        <junit.jupiter.version>5.5.1</junit.jupiter.version>
     </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.141.59</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>${junit.jupiter.version}</version>
```

Comment:
This is good. Follow this example every where for using version. By the way, I remember reading somewhere early that the junit dependency should be above all dependencies due to some resolution issues that it may otherwise land into (so above selenium dependency). But I am not completely sure. Have to find the article.